EmaPajic/assignment2/assignment2_Q1.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+@author: EmaPajic
+"""
+import unittest
+
+"""If we assume that we can have a pointer to the parent"""
```

Comment:
Good job explaining your design choices!
Please use a comment instead of a docstring when you are not commenting a class member or a function.

Reply:
updated :)